core/session/src/lib.rs

```diff
@@ -37,7 +39,8 @@ client::decl_runtime_apis! {
 	}
 }
 
-/// Generate the initial session keys with the given seeds.
+/// Generate the initial session keys with the given seeds and store them in
+/// the client's keystore.
```

Comment:
```suggestion
/// the node's keystore.
```

Reply:
It's the client's keystore, since that's the parameter to the function.